test/__tests__/data/adobe_analytics.json

```diff
@@ -2063,7 +1936,7 @@
         "JSON": {},
         "JSON_ARRAY": {},
         "XML": {
-          "payload": "<?xml version=\"1.0\" encoding=\"utf-8\"?><request><browserHeight>1794</browserHeight><browserWidth>1080</browserWidth><campaign>sales campaign</campaign><channel>web</channel><ipaddress>127.0.0.1</ipaddress><language>en-US</language><userAgent>Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)</userAgent><referrer>https://www.google.com/search?q=estore+bestseller</referrer><contextData><roott01>roottval001</roott01></contextData><eVar6>RudderLabs JavaScript SDK</eVar6><eVar2>TEDxGROWTH</eVar2><linkType>o</linkType><linkName>Watched Video</linkName><linkURL>https://www.estore.com/best-seller/1</linkURL><timestamp>2020-01-09T10:01:53.558Z</timestamp><marketingcloudorgid>mktcloudid001</marketingcloudorgid><events>event1</events><products/><reportSuiteID>footlockerrudderstackpoc</reportSuiteID></request>"
+          "payload": "<?xml version=\"1.0\" encoding=\"utf-8\"?><request><browserHeight>1794</browserHeight><browserWidth>1080</browserWidth><campaign>sales campaign</campaign><channel>web</channel><ipaddress>127.0.0.1</ipaddress><language>en-US</language><userAgent>Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)</userAgent><referrer>https://www.google.com/search?q=estore+bestseller</referrer><contextData><roott01>roottval001</roott01></contextData><eVar6>RudderLabs JavaScript SDK</eVar6><eVar2>TEDxGROWTH</eVar2><linkType>o</linkType><linkName>Watched Video</linkName><linkURL>https://www.estore.com/best-seller/1</linkURL><prop2>growth</prop2><timestamp>2020-01-09T10:01:53.558Z</timestamp><marketingcloudorgid>mktcloudid001</marketingcloudorgid><events>event1</events><products/><reportSuiteID>footlockerrudderstackpoc</reportSuiteID></request>"
```

Comment:
Why are we seeing change in the transformed payload? Is it expected?

Reply:
Previously there was no delimiter defined for property `plan` in propsDelimiter(this was probably a miss). Now we won't be having a scenario with customPropsMapping defined but not delimiter. So I added the delimiter for property `plan`, which resulted in `<prop2>growth</prop2>` being added to final payload (property `plan` has a value of growth in input event payload and mapped to index 2)

Reply:
got it